src/main/java/jpass/ui/JPassFrame.java

```diff
@@ -232,8 +275,13 @@ public static JPassFrame getInstance() {
 
     public static synchronized JPassFrame getInstance(String fileName) {
         if (instance == null) {
-            String languageTag = Configuration.getInstance().get("language.languageSetting", "en-US");
-            instance = new JPassFrame(fileName, Locale.forLanguageTag(languageTag));
+            String languageTag = Configuration.getInstance().get(LANGUAGE_LANGUAGE_SETTING, null);
```

Comment:
Why don't you simply use `Configuration.getInstance().get(LANGUAGE_LANGUAGE_SETTING, "en-US")` here?